metabrainz/templates/index/projects.html

```diff
@@ -0,0 +1,134 @@
+{% extends 'base.html' %}
+
+{% block title %}Projects - MetaBrainz Foundation{% endblock %}
+
+{% block content %}
+<h1 class="page-title">The MetaBrainz Foundation Projects</h1>
+<div id="projects">
+  
+  <div class="media">
+    <div class="media-left media-middle">
+      <a href="https://musicbrainz.org/">
+        <img class="media-object logo" src="{{ url_for('static', filename='img/projects/musicbrainz.svg') }}" alt="MusicBrainz logo">
```

Comment:
Oh, I see @Gentlecat's comment https://github.com/metabrainz/metabrainz.org/pull/208#discussion_r50640384 - I'm not convinced that my argumentation is not proper though. The names are definitely better than they were before. ;)